cmd/local/options.go

```diff
@@ -22,15 +22,22 @@ type Options struct {
 	version        string
 	profile        string
 	components     []string
+	values         []string
 }
 
-func defaultComponentList() []keb.Components {
-	defaultComponents := []string{"cluster-essentials", "istio-configuration@istio-system",
+func defaultComponents() []string {
+	return []string{"cluster-essentials", "istio-configuration@istio-system",
 		"certificates@istio-system", "logging", "tracing", "kiali", "monitoring", "eventing", "ory", "api-gateway",
 		"service-catalog", "service-catalog-addons", "rafter", "helm-broker", "cluster-users", "serverless",
 		"application-connector@kyma-integration"}
-	return componentsFromStrings(defaultComponents)
-
+}
+func defaultValues() []string {
+	return []string{
+		"tracing.authProxy.config.useDex=false",
+		"tracing.authProxy.configDocsLink=https://kyma-project.io/docs",
+		"kiali.authProxy.config.useDex=false",
+		"kiali.authProxy.configDocsLink=https://kyma-project.io/docs",
+	}
```

Comment:
Such information should go into the particular component and not become part of the reconciler.

Reply:
Removed.